### PR TITLE
Fixes #506: Add MyRatings Card component to dashboard page

### LINKED ISSUE #506: Add a profile page where each user can see the skills they have created and get aggregated stats
**Actual Behaviour**
No profile page is present for logged in user

**Expected Behaviour**
Once the user is logged in we can have a page where the user can see the skills they created and several aggregated statistics on those skills like total upvotes, downvotes, average rating, total users that rated their skill etc.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by anshumanv**
Fixes #506 

Changes: Added a card which lists all the ratings the logged in user has given to all skills

Surge Deployment Link: https://pr-777-fossasia-susi-skill-cms.surge.sh

Screenshots for the change:
image

### CODE REVIEW

File: src/components/Dashboard/MyRatings.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+        )}
+        {ratingsData.length === 0 &&
+          !this.state.loading && (
+            <div>
+              <div className="center">
+                <br />
+                <h2>
+                  You have not rated any skill, go to{' '}
+                  <a
+                    href="https://github.com/fossasia/susi_skill_cms/blob/master/docs/Skill_Tutorial.md"
```
Comment (Akshat-Jain) [inline src/components/Dashboard/MyRatings.js]: I'm not sure if this should redirect to the Skill Tutorial file. We can't rate any skill there. How is that related here? Let's wait for more opinions anyways. Rest LGTM :+1:

Comment (anshumanv) [reply src/components/Dashboard/MyRatings.js]: Ah sorry, I took that from MySkills and forgot to replace the link, that should redirect to index page.

Comment (anshumanv) [reply src/components/Dashboard/MyRatings.js]: Fixed
